### PR TITLE
Docs: add stackblitz to new component

### PR DESCRIPTION
**Pull request by stefanoScalzo**
#### Please provide a link to the associated issue.
fixes: #1426 
fixes: #1379

#### Please provide a brief summary of this pull request.
Uploaded the documentation for new component creation by adding a link for the stackblitz instructions
Added proper identifiers to each example file, modified the files by changing some inline scss to seperate scss files. Also separated some example .ts files. Added the stackblitz to the code-example.ts file

### CODE REVIEW

File: apps/docs/src/app/documentation/core-helpers/code-example/code-example.component.ts [hunk 1]
```diff
@@ -32,23 +37,231 @@ export class CodeExampleComponent implements OnInit, AfterViewInit {
...
+                this.project.files['src/app/app.module.ts'] = example.code.default;
+            }
+        });
+
+        this.project.files['src/index.html'] = `
+        <link rel="stylesheet" href="node_modules/fundamental-styles/dist/fonts.css"></link>
+        <link rel="stylesheet" href="node_modules/fundamental-styles/dist/icon.css"></link>
+                    <${this.parameters.html_tag}></${this.parameters.html_tag}>
+        `;
+        // TODO make non inline
```
Comment (droshev) [inline apps/docs/src/app/documentation/core-helpers/code-example/code-example.component.ts]: todo what?

**Comment by droshev** [comment]
looks like `angular.json` is now being loaded; nevertheless it works without it so we might not need it

File: apps/docs/src/app/documentation/core-helpers/code-example/code-example.component.ts [hunk 2]
```diff
@@ -32,23 +37,231 @@ export class CodeExampleComponent implements OnInit, AfterViewInit {
...
+    project = {
+        files: {},
+        title: 'Fundamental-NGX Example',
+        description: 'Generated for you by fundamental-ngx team',
+        template: 'angular-cli',
+        tags: ['stackblitz', 'sdk'],
+        dependencies: {
+            moment: '*',
+            '@fundamental-ngx/core': 'v0.12.0-rc.45',
+            'fundamental-styles': 'v0.3.0-rc.6',
```
[the remark this answers is not in the document]
Comment (stefanoScalzo) [reply apps/docs/src/app/documentation/core-helpers/code-example/code-example.component.ts]: Fixed

File: apps/docs/src/app/documentation/core-helpers/code-example/code-example-stack/main.ts [hunk 3]
```diff
@@ -0,0 +1,6 @@
+// import { enableProdMode } from '@angular/core';
```
Comment (droshev) [inline apps/docs/src/app/documentation/core-helpers/code-example/code-example-stack/main.ts]: if this is commented, is it used or does it make sense to be included?

File: libs/core/src/lib/table/table-wrapper.component.ts [hunk 4]
```diff
@@ -20,7 +20,7 @@ export class TableWrapperComponent implements AfterContentInit {
 
     constructor(
         private elementRef: ElementRef
-    ) {}
+    ) { }
```
Comment (droshev) [inline libs/core/src/lib/table/table-wrapper.component.ts]: can you revert this?

File: libs/core/src/lib/datetime-picker/datetime-picker.component.ts [hunk 5]
```diff
@@ -184,7 +184,7 @@ export class DatetimePickerComponent implements OnInit, OnDestroy, ControlValueA
      * @param fdDate FdDate
      */
     @Input()
-    disableFunction = function(fdDate: FdDate): boolean {
+    disableFunction = function (fdDate: FdDate): boolean {
```
Comment (droshev) [inline libs/core/src/lib/datetime-picker/datetime-picker.component.ts]: can you revert this file?

File: tslint.json [hunk 6]
```diff
@@ -1,151 +1,100 @@
...
-    "member-access": false,
-    "member-ordering": [
-      true,
-      {
-        "order": [
-          "static-field",
-          "instance-field",
-          "static-method",
-          "instance-method"
+    "extends": "tslint:recommended",
```
Comment (droshev) [inline tslint.json]: did you modify this file?

File: apps/docs/src/app/core/documentation/core-documentation.component.html [hunk 7]
```diff
@@ -1,19 +1,17 @@
 <button (click)="skipNavClicked()" aria-label="skip navigation" class="skip-navigation">Skip Navigation</button>
 <fd-docs-toolbar (btnClicked)="handleMenuCollapseClick()"></fd-docs-toolbar>
 <div class="fd-flex-row fd-sidebar-container">
-    <sections-toolbar
-        [sections]="sections"
-        [(sideCollapsed)]="sideCollapsed"
-    ></sections-toolbar>
+    <sections-toolbar [sections]="sections" [(sideCollapsed)]="sideCollapsed"></sections-toolbar>
```
Comment (droshev) [inline apps/docs/src/app/core/documentation/core-documentation.component.html]: the changes in this file can be done in a separate PR

File: apps/docs/src/app/core/core-documentation.module.ts [hunk 8]
```diff
@@ -131,9 +131,10 @@ import { ModalOpenTemplateExampleComponent } from '../core/component-docs/modal/
 import { ModalContentComponent } from '../core/component-docs/modal/examples/component-as-content/modal-content.component';
 import { ModalComponentAsContentExampleComponent } from '../core/component-docs/modal/examples/component-as-content/modal-component-as-content-example.component';
 import {
-    PanelEdgeBleedExampleComponent,
     PanelExampleComponent
 } from '../core/component-docs/panel/examples/panel-examples.component';
+import { PanelEdgeBleedExampleComponent } from '../core/component-docs/panel/examples/panel-edge-bleed-example.component'
```
Comment (droshev) [inline apps/docs/src/app/core/core-documentation.module.ts]: I guess this file was just auto-formatted - if so then we can do that in a separate PR

File: apps/docs/src/app/documentation/core-helpers/code-example/code-example.component.ts [hunk 9]
```diff
@@ -32,23 +35,227 @@ export class CodeExampleComponent implements OnInit, AfterViewInit {
     @Input()
     exampleFiles: ExampleFile[] = [];
 
+    // @Input() fileName: string;
```
Comment (droshev) [inline apps/docs/src/app/documentation/core-helpers/code-example/code-example.component.ts]: do we need those?

Comment (stefanoScalzo) [reply apps/docs/src/app/documentation/core-helpers/code-example/code-example.component.ts]: yes we do there passed in each docs files
